fancylog/fancylog.py

```diff
@@ -60,7 +61,8 @@ def start_logging(
     :param log_to_file: If True, write a log file, otherwise just print to
     terminal.
     :param timestamp: If True, add a timestamp to the filename
-    :return: Path to the logging file
+    :param log_to_console: Print logs in stderr or not: Default: True
```

Comment:
Would this not be to stdout?

Reply:
that's correct, I renamed the function as realised this but forgot to update the doc, thanks!